projects/scion/workbench-application.core/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": true
```

Comment:
This module is a pure *TypeScript* module and does not depend on Angular. To simplify the build, it is built with `ng-packagr`. Please remove the `angularCompilerOptions` section here, but keep the file for consistency reasons.

Reply:
Done